samples/templates/default-azuredeploy-docker.json

```
@@ -319,6 +312,12 @@
                 "reserved": true
             }
         },
+        {
+            "type": "Microsoft.ManagedIdentity/userAssignedIdentities",
+            "apiVersion": "2018-11-30",
+            "name": "[variables('appServicePlanName')]",
```

I think we should create a new variable for the UAMI name, and I think align with the DB name e.g.

```
"managedIdentityName": "[concat(if(empty(parameters('sqlServerName')),variables('serviceName'),parameters('sqlServerName')), '-uami')]",
```

then here use:

```suggestion
            "name": "[variables('managedIdentityName')]",
```